OpenGL/Constructs/VAO.cs

```diff
@@ -578,7 +597,8 @@ private void DrawOGL3()
         {
             if (ID == 0 || VertexCount == 0) return;
             Gl.BindVertexArray(ID);
-            Gl.DrawElements(DrawMode, VertexCount, elementType, IntPtr.Zero);
+            IntPtr offset = (IntPtr)(Offset * GetElementSizeInBytes());
+            Gl.DrawElements(DrawMode, VertexCount, elementType, offset);
```

Comment:
Similar to the comment above, it doesn't have to be for this commit, but I think it might be worth caching the computation of offset (especially since I imagine offset is usually zero) since this happens in the Draw code.

Reply:
It could be computed as part of the setter of the Offset property, backed by two private fields?  Curious on your thoughts.